### ISSUE TITLE
Plugin Unavailable on Windows Use 13.6 Version

### ISSUE THREAD

**Issue by baronliu** [gitkraken/vscode-gitlens#2735]
### Description

I can use VS Code Remote to access a remote project on my Mac computer with version 13.6, but when I perform the same operation on a Windows computer, it doesn't work. The plugin page is also grayed out. However, if I roll back to version 13.5, it works fine.

### GitLens Version

v13.6.0

### VS Code Version

版本: 1.75.0 (user setup)
提交: e2816fe719a4026ffa1ee0189dc89bdfdbafb164
日期: 2023-02-01T15:23:45.584Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.19045
沙盒化: No

### Git Version

git version 2.9.5

### Logs, Screenshots, Screen Captures, etc

![image](https://github.com/gitkraken/vscode-gitlens/assets/15775119/43772408-882c-43b5-9afb-f6c3aa30f388)

**Comment by eamodio** [comment]
@baronliu What does it say when you hover over the disabled/grayed out entry for GitLens in the extensions view?

**Comment by baronliu** [comment]
@eamodio  It looks normal. This is a brief description of the plugin.
<img width="938" alt="企业微信截图_1686632734985" src="https://github.com/gitkraken/vscode-gitlens/assets/15775119/ca43974e-7a3e-45f6-8015-9a48baab466e">
![image](https://github.com/gitkraken/vscode-gitlens/assets/15775119/66c6c0b4-7b30-47da-bdb1-04bb2544528f)

**Comment by baronliu** [comment]
I noticed a phenomenon, and I'm not sure if it's helpful for solving the problem. When I uninstall the plugin on my Windows computer and reinstall it, the plugin gets reinstalled with version v13.5.0, and then the issue is resolved. However, when I go back home and use my Mac computer, the automatic update updates the plugin to v13.6.0. Since I am doing remote development, when I return to my Windows computer, the problem reappears.

**Comment by eamodio** [comment]
Hrm, are the versions of VS Code the same on both machines?

**Comment by baronliu** [comment]
@eamodio 
Thank you, I think I've found the problem. Because VS Code on my Mac is set to automatic updates, it's at version 1.79.0, while on Windows, it's at version 1.75.0. It seems that the incompatibility is caused by the mismatch between version 1.75.0 and the v13.6.0 plugin. After manually upgrading VS Code on my Windows computer, the issue was resolved. It appears that the automatic update feature on my Windows computer didn't take effect. I'll investigate this separate issue further later.
